electron/constants.ts

```diff
@@ -107,11 +108,22 @@ function fixAsarPath(origin: string): string {
   return origin
 }
 
-function getSteamCompatFolder() {
-  if (existsSync(`${userHome}/.var/app/com.valvesoftware.Steam/.steam/steam`)) {
-    return `${userHome}/.var/app/com.valvesoftware.Steam/.steam/steam`
+export function getSteamCompatFolder() {
+  // Paths are from https://savelocation.net/steam-game-folder
+  if (isWindows) {
+    return join('C:/Program Files (x86)/Steam')
```

Comment:
Atually we need to use normalize from path. Let path care about it.

Reply:
Are you sure normalize does anything for us here?
```
> path.normalize('C:/Program Files (x86)/Steam')
'C:/Program Files (x86)/Steam'
```

Reply:
Yes exactly. It is the same as in Python. We should start using the node Path object instead of strings.
In Python i learned to use always Path type instead of strings.